extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -214,6 +214,22 @@ and the user can select a date range with a starting date and ending date.
   </noscript>
 </amp-img>
 
+## Date formats
+
+`amp-date-picker` attributes accept dates in ISO 8601 and RFC 5545 RRULE formats.
+
+ISO 8601 formats dates as YYYY-MM-DD and is the standard for sharing dates between
+electronic systems. For example, ISO 8601 formats the date February 28 2018 as
+`2018-02-28`.
+
+RFC 5545 standardizes a format called RRULE for specifying repeating dates.
+For example, RFC 5545 formats Halloween as `RRULE:FREQ=YEARLY;BYMONTH=10;BYMONTHDAY=31`.
+More complex dates are also possible, such as the United States Thanksgiving holiday,
+which is every November on the fourth Thursday: `RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=+4TH`.
+The API is not friendly to memorize, but there are various
```

Comment:
Is there documentation on the API we can link out to?

Reply:
Good call, that is definitely important here. I'll add links to those